src/binary/component.rs

```diff
@@ -17,10 +17,25 @@ pub enum ComponentType {
 #[cfg(any(feature = "backend-kvm"))]
 pub const PT_ENARX_SALLYPORT: u32 = PT_LOOS + 0x34a0001;
 
-/// The enarx code program header type
-#[cfg(any(feature = "backend-kvm"))]
+/// The enarx init program header type
```

Comment:
"init"? .. ah, you mean PID 1 .. the only unikernel process. "init" is a little bit ambiguous.

Reply:
Ugh. Typo.

Reply:
Fixed.